src/WebJobs.Script.WebHost/Properties/launchSettings.json

```diff
@@ -4,9 +4,17 @@
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "AzureWebJobsScriptRoot": "D:\\apps\\HelloHttpNet9NoProxy",
```

Comment:
Was this done inadvertently?

Reply:
Accidently included that file. Removed in the latest iteration.